src/Servers/IIS/src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -20,8 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="AspNetCoreModule" Condition="'$(OS)' == 'Windows_NT'" />
-    <Reference Include="AspNetCoreModuleV2" Condition="'$(OS)' == 'Windows_NT'" />
+    <Reference Include="AspNetCoreModule" Condition="'$(OS)' == 'Windows_NT' AND '$(VCTargetsPath)' != ''" />
```

Comment:
So, this effectively skips this dependency when using `dotnet build`. Is that the behavior we want? Just running `dotnet test` will fail, right?

cc @pakrym

Reply:
Having a full build is still required, `dotnet test` would work if native assets were pre-built.

Reply:
We can add an error conditions that would fail during `Microsoft.AspNetCore.Server.IIS.csproj` project build and say that native components have to be build using `.\build`

Reply:
👍 that's a good idea.